worlds/witness/docs/setup_fr.md

```diff
@@ -0,0 +1,47 @@
+# Guide d'installation du Witness randomizer
+
+## Logiciels Requis
+
+- [The Witness pour Windows 64-bit (par exemple, la version Steam)](https://store.steampowered.com/app/210970/The_Witness/)
+- [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+
+## Logiciels Facultatifs
+
+- [ArchipelagoTextClient](https://github.com/ArchipelagoMW/Archipelago/releases)
+- [The Witness Map- et Auto-Tracker](https://github.com/NewSoupVi/witness_archipelago_tracker/releases), pour usage avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+## Rejoindre un jeu multimonde
+
+1. Lancez The Witness
+2. Commencez une nouvelle partie
+3. Lancez [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+4. Inscrivez l'adresse Archipelago, votre nom de joueur et le mot de passe du jeu multimonde
+5. Cliquez sur "Connect"
+6. Jouez!
+
+Pour continuer un jeu multimonde précedemment commencé:
+
+1. Lancez The Witness
+2. Chargez la sauvegarde sur laquelle vous avez dernièrement joué ce monde, si ce n'est pas celle qui a été chargée automatiquement 
+3. Lancez [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+4. Cliquez sur "Load Credentials" (ou tapez les manuellement)
+5. Cliquez sur "Connect"
+
+## Archipelago Text Client
+
+Il est recommandé d'utiliser le "Archipelago Text Client" en parallèle afin de suivre quels items vous envoyez et recevez.
+<br/>The Witness affiche également ces informations en jeu, mais seulement pour une courte période et donc il est facile de manquer ces messages.
+
+<br/><br/>Bien sûr, vous pouvez également utiliser l'auto-tracker!
+
+## Auto-Tracking
+
+The Witness a un tracker fonctionnel qui supporte l'auto-tracking.
+
+1. Téléchargez [The Witness Map- and Auto-Tracker](https://github.com/NewSoupVi/witness_archipelago_tracker/releases) et [PopTracker](https://github.com/black-sliver/PopTracker/releases).
+2. Ouvrez Poptracker, puis chargez le pack Witness. 
+3. Cliquez sur l'icone "AP" qui se situe au dessus de la carte.
+4. Inscrivez l'adresse Archipelago, votre nom de joueur et le mot de passe du jeu multimonde.
+
+Le reste devrait être pris en charge par Poptracker - les items que vous recevrez et les puzzles que vous résolverez seront automatiquement indiqués. De plus, Poptracker est en mesure de détecter
+vos paramètres de jeu - les puzzles accessibles seront alors masqués ou affichés en fonction de vos paramètres de randomization et de logique. Veuillez noter que le tracker peut être obsolète.
```

Comment:
I think the note about the poptracker being out of date can probably be removed? It doesn't seem to appear to be in the english version (anymore?)

Otherwise, I'd probably word it more along the lines of "Veuillez noter que le pack peut demander une mise à jour". Obsolete sounds a too final.